multiqc/templates/default/includes.html

```diff
@@ -32,6 +32,9 @@
     {{ include_file('assets/css/default_multiqc.css') }}
     {{ include_file('assets/css/jquery.toast.css') }}
 </style>
+{% for css_href in config.custom_css_files %}
+<style type="text/css">{{ include_file(css_href, None) }}</style>
```

Comment:
I forget, what is the second argument for? Is it needed?

Reply:
```
    def include_file(name, fdir=tmp_dir, b64=False):
        try:
            if fdir is None:
                fdir = ""
            if b64:
                with io.open(os.path.join(fdir, name), "rb") as f:
                    return base64.b64encode(f.read()).decode("utf-8")
            else:
                with io.open(os.path.join(fdir, name), "r", encoding="utf-8") as f:
                    return f.read()
```

It's the root dir, looks like. I suspect we can actually omit it. I just copy/pasta'd from right above it.